addons/website_slides_survey/views/website_profile.xml

```diff
@@ -13,7 +13,8 @@
         <xpath expr="//ul[hasclass('o_wprofile_nav_tabs')]" position="inside">
             <li t-if="show_certification_tab" class="nav-item">
                 <a role="tab" aria-controls="certification" href="#profile_tab_content_certification" t-attf-class="nav-link #{ 'active' if active_tab == 'certification' else '' }" data-bs-toggle="tab">
-                    <t t-if="user_inputs" t-out="len(user_inputs)" /> Certification Attempts
+                    <t t-if="user_inputs" t-set="certification_attempts" t-value="len([user_input for user_input in user_inputs if user_input['state'] == 'done'])"/>
```

Comment:
we could remove [ and ] to have a generator

```suggestion
                    <t t-if="user_inputs" t-set="certification_attempts" t-value="len(user_input for user_input in user_inputs if user_input['state'] == 'done')"/>
```
(same things below)

Reply:
> there is some risk with changing the XML structure like this

Indeed. How about upating `WebsiteSlidesSurvey._prepare_user_profile_values` to add `('state', '=', 'done')` in the domain instead? I don't think we ever need to show the other attempts.

> note: if in all case we wanted them filtered, maybe it mike make sense to already give the list filtered to the view

= this ^^